CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 All notable changes to the Zowe API Mediation Layer package will be documented in this file.
 
+## `1.24.4 (2021-09-01)`
+
+* Feature:  SAF resource access check controller (#1734) ([680f4b9](https://github.com/zowe/api-layer/commit/680f4b9)), closes [#1734](https://github.com/zowe/api-layer/issues/1734)
+* Feature:  compress response if service wants to (#1728) ([85b5948](https://github.com/zowe/api-layer/commit/85b5948)), closes [#1728](https://github.com/zowe/api-layer/issues/1728)
```

Comment:
Is this accomplished via GZIP compression? We should specify how this compression is performed.

Reply:
tweaked